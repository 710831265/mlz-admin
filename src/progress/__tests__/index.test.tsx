import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import Progress from '..';

describe('ðŸ§ª Progress', () => {
  testMount(Progress);
  testSnapshot(Progress);
});
