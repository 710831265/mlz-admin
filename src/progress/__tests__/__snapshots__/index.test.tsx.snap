// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Progress snapshot test snapshot comparing 1`] = `"<div class=\\"ant-progress ant-progress-line ant-progress-status-normal ant-progress-show-info ant-progress-default\\"><div class=\\"ant-progress-outer\\"><div class=\\"ant-progress-inner\\"><div class=\\"ant-progress-bg\\" style=\\"width: 0%; height: 8px;\\"></div></div></div><span class=\\"ant-progress-text\\" title=\\"0%\\">0%</span></div>"`;
